modules/rostests/apitests/kernel32/SetUnhandledExceptionFilter.c

```diff
@@ -96,11 +113,19 @@ VOID TestSSEExceptions(VOID)
     p = SetUnhandledExceptionFilter(ExceptionFilterSSESupport);
     ok(p == NULL, "Previous filter should be NULL\n");
 #ifdef _MSC_VER
+#if defined(_M_AMD64)
+    {
+        __m128 xmm = { { 0 } };
+        xmm = _mm_xor_ps(xmm, xmm);
```

Comment:
Can't it work for other architectures/compilers too?

Reply:
Only if someone implements this stuff for GCC. It doesn't work on GCC with our type definitions, so we should probably import the mingw header instead.